doc/cephfs/kernel.rst

```diff
@@ -31,4 +31,4 @@ To unmount the Ceph file system, you may use the ``umount`` command. For example
 See `mount.ceph`_ for details.
 
 .. _mount.ceph: ../../man/8/mount.ceph/
-.. _Authentication: ../../rados/operations/authentication/
\ No newline at end of file
+.. _Authentication: ../../rados/operations/user-management/
```

Comment:
might want to update the place referencing this section as well to reflect the updated section name of the "Authentication" section.